tests/components/StandardFacets.test.tsx

```diff
@@ -75,6 +75,17 @@ describe('StandardFacets', () => {
     expectFacetOptionSet(actions, productFacet.fieldId, productFacet.options[0], true);
   });
 
+  it('does not display option counts if setOptionCounts is set to false', () => {
+    render(<StandardFacets showOptionCounts={false}/>);
+
+    const facets = DisplayableFacets[0];
+    const coffeeLabel = screen.queryByLabelText(facets.options[0].displayName);
+    const coffeeCount = screen.queryByLabelText(facets.options[0].count);
+
+    expect(coffeeLabel).toBeDefined();
+    expect(coffeeCount).toBeNull();
```

Comment:
I believeee this assertion wouldn't help since screen.queryByLabelText doesn't do for partial text match, so `stuff (0)` vs `stuff` for the label would still pass for this assertion since it will always be null. You may want to build the string as if showOptionCounts is true and check that instead

Reply:
OK! I changed it for safely-- but I think that queryByLabelText does do partial text match!